model/task/task.go

```diff
@@ -324,14 +325,70 @@ func (tr *TestResult) GetLogTestName() string {
 
 // GetDisplayTestName returns the name of the test that should be displayed in
 // the UI. In most cases, this will just be TestFile.
-func (tr *TestResult) GetDisplayTestName() string {
+func (tr TestResult) GetDisplayTestName() string {
 	if tr.DisplayTestName != "" {
 		return tr.DisplayTestName
 	}
 
 	return tr.TestFile
 }
 
+// GetLogURL returns the external or internal log URL for this test result.
+func (tr TestResult) GetLogURL(raw bool) string {
+	root := evergreen.GetEnvironment().Settings().ApiUrl
+	deprecatedLobsterURL := "https://logkeeper.mongodb.org"
```

Comment:
Should this be renamed to `deprecatedLogkeeperURL`? Lobster is just the front end for viewing HTML logs.

Reply:
Good catch

Reply:
Actually, this is correct. Lobster was hosted in previously hosted in logkeeper and we moved it over to evergreen.

Reply:
Sure but were the raw logs also using lobster?

Reply:
Yeah, I replace it for raw logs as well.

Reply:
My question is about the variable name - lobster isn't used at all for raw logs is it? That would make this a deceptive field by implying that lobster is somehow tied to the raw logs.

Reply:
Ah, yes I see what you are saying now. This is actually a bug—raw logs from logkeeper still need to use the `logkeeper.mongodb.org` route. Thanks for the pushback!

Reply:
But I still think the variable name should be `deprecatedLobsterURL` cause `logkeeper.mongodb.org` itself is not deprecated (we still download valid logs from there) it more so that we no longer want to use the lobster hosted there.